android-resources/www/css/index.css

```diff
@@ -0,0 +1,105 @@
+* {
```

Comment:
Please add a source header, which you can copy from other css files in the project (e.g., in web/css).